test/integration/nearcache/RepairingTaskTest.js

```diff
@@ -57,7 +57,8 @@ describe('RepairingTask', function () {
     }
 
     it('throws when reconciliation interval is set to below 30 seconds', async function () {
-        return expect(startClientWithReconciliationInterval( 2)).to.be.eventually.rejected;
+        await startClientWithReconciliationInterval(2);
```

Comment:
reverted the change in this test to its version in master branch. I changed it before due to `RepairingTask` construction change.